OpenSim/Simulation/Model/Muscle.cpp

```diff
@@ -144,6 +157,7 @@ void Muscle::constructProperties()
     // By default the min and max controls on muscle are 0.0 and 1.0
     setMinControl(0.0);
     setMaxControl(1.0);
+    upd_GeometryPath().setDefaultColor(DefaultMuscleColor);
```

Comment:
Good point @chrisdembia.  I haven't explicitly tested (or looked for models with PathSpring, ClutchSpring, Ligament and possibly other kind of Objects that use GeometryPath). I only changed the default colors for those objects proposed by reviewers. If all works for Muscles (which is a new functionality anyway) then we can apply the same for other objects as needed. I'd prefer to have models to test with before making changes. If you think I  should hold until we have such models or you can find example models I'd be happy to make changes/test.